client/src/components/videopanel/VideoPanel.jsx

```diff
@@ -16,7 +17,9 @@ export default function VideoPanel() {
   const onSubmit = async (uploadData) => {
     try {
       await axios.post(`${expressURL}/api/videos`, uploadData);
+      toast.success("video added successfully!");
     } catch (err) {
+      toast.error("an error occured, please try again");
```

Comment:
again: `an error` -> `An error`
the message _might_ "flow" better with: `An error has occured, please try again later` (do as you see fit, either can work)